kimsoohyun/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express')
+const logger = require('morgan');
+const cors = require('cors')
+const app = express()
+const dotenv = require("dotenv")
+const { DataSource } = require('typeorm');
+
+dotenv.config()
```

Comment:
```suggestion
require("dotenv").config();

const express = require ("express");
const cors = require ("cors");
const morgan = require ("morgan");
```
1. dotenv 같이 서버 구동에 필요한 환경변수를 읽어 오는 코드는 최상단에 적어주세요.
2. dotenv와 관련된 코드는 위와 같이 한줄로 작성할 수도 있습니다. 한 번 기존 코드와 비교해보세요!
3. express 를 활용하기 때문에 http 모듈을 쓸 필요가 없습니다. express가 http(네트워크 통신에 대한 처리를 도와주는 node.js 빌트인 모듈) 대신에 보다 더 많은 기능을 제공합니다.
4. 코드 순서 예시 참고하여 수정부탁드릴게요~